enzyme/Enzyme/Utils.h

```diff
@@ -1051,7 +1051,11 @@ template <typename T> static inline llvm::StringRef getFuncNameFromCall(T *op) {
 }
 
 template <typename T>
+#if LLVM_VERSION_MAJOR >= 16
+static inline std::optional<size_t> getAllocationIndexFromCall(T *op) {
```

Comment:
Can you put the curly braces after the ifdef? It confuses some editors otherwise.

Reply:
done